lib/js_of_ocaml/js.ml

```diff
@@ -843,6 +843,12 @@ let export_all obj =
 
 (****)
 
+module Effect = struct
```

Comment:
`Jsoo_runtime` feels like an internal library. It does not seem to be documented anywhere. But maybe we should change that?

Reply:
Jsoo_runtime was created so that alternative lib (such as brr and gen_js_api) can rely on jsoo primitives without having to depend on the whole jsoo lib. The module should be re-exported and documented in js_ of_ocaml, the lib.